comps/reranks/README.md

```diff
@@ -0,0 +1,31 @@
+# Reranking Microservice
+
+The Reranking Microservice, fueled by reranking models, stands as a straightforward yet immensely potent tool for semantic search. When provided with a query and a collection of documents, reranking swiftly indexes the documents based on their semantic relevance to the query, arranging them from most to least pertinent. This microservice significantly enhances overall accuracy. In a text retrieval system, either a dense embedding model or a sparse lexical search index is often employed to retrieve relevant text documents based on the input. However, a reranking model can further refine this process by rearranging potential candidates into a final, optimized order.
+
+![Flow Chart](./assets/img/reranking_flow.png)
+
+---
+
+## 🛠️ Features
+
+- **rerank on retrieved documents**: Perform reranking on the given documents using reranking models together with query.
+
+---
+
+## ⚙️ Implementation
```

Comment:
It would be nice to have some description to guide people what each of these are for and why they would choose one over another.

Reply:
hi @dcmiddle ,
There's description/explanation on each of them when user click into each of the respective README file for each implementation. I'm trying to keep this root README clean and simple to let user have quick read and understanding what this component is through this README.

Reply:
Hi @dcmiddle ,
Ya I have the same concern with you where this videoqna is fairly new to me compared to other three. Where im also abit confuse on this, from what i see it is more specific to a use case of videoqna using this method. Hence i did not explicitly modify anything on that readme itself rather than just reformatting it without changing the original content. 
Next for your concern of "eople who aren't experts can compose what they need." I think we can come out with a standard way of compiling the comps root folder readme thru some discussion. Or someone have better idea of having it in a clean and understandable view would be great. For now, my main goal in this PR was to address to this request https://github.com/opea-project/GenAIComps/issues/726 and standardize it with other components.

Reply:
For videoqna readme wise, we can request the owner to refine the readme guide again to include a better background for understanding of that implementation.